src/Functions/byteSwap.cpp

```diff
@@ -62,46 +62,39 @@ struct FunctionUnaryArithmeticMonotonicity<NameByteSwap>
 
 REGISTER_FUNCTION(ByteSwap)
 {
-    factory.registerFunction<FunctionByteSwap>(
-        FunctionDocumentation{
-            .description = R"(
+    FunctionDocumentation::Description description = R"(
 Reverses the bytes of an integer, i.e. changes its [endianness](https://en.wikipedia.org/wiki/Endianness).
 
-**Example**
+The below example can be worked out in the following manner:
 
-```sql
-byteSwap(3351772109)
-```
-
-Result:
-
-```result
-┌─byteSwap(3351772109)─┐
-│           3455829959 │
-└──────────────────────┘
-```
-
-The above example can be worked out in the following manner:
 1. Convert the base-10 integer to its equivalent hexadecimal format in big-endian format, i.e. 3351772109 -> C7 C7 FB CD (4 bytes)
 2. Reverse the bytes, i.e. C7 C7 FB CD -> CD FB C7 C7
-3. Convert the result back to an integer assuming big-endian, i.e. CD FB C7 C7  -> 3455829959
-
-One use-case of this function is reversing IPv4s:
+3. Convert the result back to an integer assuming big-endian, i.e. CD FB C7 C7 -> 3455829959
+One use case of this function is reversing IPv4s:
 
 ```result
 ┌─toIPv4(byteSwap(toUInt32(toIPv4('205.251.199.199'))))─┐
 │ 199.199.251.205                                       │
 └───────────────────────────────────────────────────────┘
 ```
-)",
-            .examples{
-                {"8-bit", "SELECT byteSwap(54)", "54"},
-                {"16-bit", "SELECT byteSwap(4135)", "10000"},
-                {"32-bit", "SELECT byteSwap(3351772109)", "3455829959"},
-                {"64-bit", "SELECT byteSwap(123294967295)", "18439412204227788800"},
-            },
-            .category = FunctionDocumentation::Category::Arithmetic},
-        FunctionFactory::Case::Insensitive);
+    )";
+    FunctionDocumentation::Syntax syntax = "byteSwap(x)";
+    FunctionDocumentation::Argument argument1 = {"x", "An integer value."};
+    FunctionDocumentation::Arguments arguments = {argument1};
+    FunctionDocumentation::ReturnedValue returned_value = "x with bytes reversed";
+    FunctionDocumentation::Example example1 = {"", "byteSwap(3351772109)", R"(
+┌─byteSwap(3351772109)─┐
```

Comment:
Could we add `SELECT ` in front of `byteSwap(3351772109)` and remove the formatting around the result (l. 86-88, i.e. just have `3455829959` as a result)?